File: Hari02/Js/Lingkaran.js
const hitungLingkaran = (radius) => {
    const luas = Math.PI * Math.pow(radius, 2);
    const keliling = 2 * Math.PI * radius;
    return { luas, keliling };
}

const radius = parseFloat(prompt("Masukkan jari-jari lingkaran:"));
const { luas, keliling } = hitungLingkaran(radius);
console.log("Luas:", luas);
console.log("Keliling:", keliling);

File: Hari02/Python/Lingkaran.java
import java.util.Scanner;

public class Lingkaran {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Masukkan jari-jari lingkaran: ");
        double radius = input.nextDouble();

        double luas = Math.PI * Math.pow(radius, 2);
        double keliling = 2 * Math.PI * radius;

        System.out.println("Luas :" + luas);
        System.out.println("Keliling :" + keliling);
    }
}
